client/src/router/index.ts

```diff
@@ -38,10 +47,10 @@ bikeVersions.forEach((v) => {
 /**
  * Get the route info for the selected bike version
  *
- * @returns {VersionInfo} Bike version route info
+ * @returns {VersionInfo | null} Bike version route info
```

Comment:
Don't need to duplicate the type information

Reply:
Done